src/components/upload-file/index.js

```diff
@@ -1,43 +1,81 @@
 import React, { useState } from "react";
 import styles from './index.module.scss';
+import stylesDropDawn from './drop-dawn.module.scss';
+import { Link } from "react-router-dom";
+
+import StepProgress from "../step_progress";
 import '../../fontawesome-free/css/all.css';
 import { ReactComponent as IconUploadFile } from './icon-upload-file.svg';
 import { ReactComponent as IconCart } from './icon-cart.svg';
+import { ReactComponent as IconArrow } from './icon-arrow.svg';
 
 const UploadFileComponent = () => {
     const [file, setFile] = useState("");
+    const [selectStep] = useState(2);
+    const [dropDawn, setDropDawn] = useState(5);
+
+    const handleChangeDropDawn = (e) => {
+        console.log("e>>>", e.value)
+        setDropDawn(e.value);
+    };
+
     const handleChange = event => {
-        setFile(URL.createObjectURL(event.target.files[0]))
+        if (event.target.files) {
+            setFile(URL.createObjectURL(event.target.files[0]))
+        }
     }
+
     return (
-        <main className={styles.wrapContent}>
-            <img src={file} className={styles.square} />
-
-            <div className={styles.rightContent}>
-                <label>อนุมัติรูปแบบงาน</label>
-                <div className={styles.boxInput}>
-                    <i class="fas fa-check"></i>
-                    <select>
-                        <option>
-                            <i class="fas fa-check"></i> 
-                            ต้องการอนุมัติ</option>
-                        <option>ไม่ต้องการอนุมัติ</option>
-                    </select>
-                </div>
+        <main>
+            <section style={{ display: "flex" }}>
+                <StepProgress stepIndex={selectStep} />
+            </section>
+            <section className={styles.wrapContent}>
+                <img src={file} className={styles.square} />
+
+                <div className={styles.rightContent}>
+                    <label className={styles.label}>อนุมัติรูปแบบงาน</label>
 
-                <label>อัพโหลดไฟล์งาน</label>
+                    <div className={stylesDropDawn.selectBox}>
+                        <div className={stylesDropDawn.selectBoxCurrent} tabindex="1">
+                            <div className={stylesDropDawn.selectBoxValue}>
+                                <input className={stylesDropDawn.selectBoxInput} type="radio" id="1" value="2" name="Ben" checked={dropDawn == 2 ? true : false}
+                                    onChange={(e) => handleChangeDropDawn(e.target)} />
```

Comment:
why the hell is everything dropdawn

Reply:
if u want to merge stylesDropDawn in styles. i can do it